.github/workflows/busola-web-build.yml

```diff
@@ -36,6 +41,5 @@ jobs:
     with:
       name: busola-web
       dockerfile: Dockerfile.web
-      context: .
-      export-tags: true
-      tags: latest
+      tags: ${{ inputs.tag != '' && inputs.tag || 'latest' }}
+      build-args: ${{ inputs.tag != '' && format('tag={0}', inputs.tag) || '' }}
```

Comment:
Make sure after first release on dev, that it is working and displayed correctly